staging/src/github.com/kubeedge/beehive/pkg/core/socket/context_socket.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/kubeedge/beehive/pkg/core/socket/wrapper"
 )
 
+//nolint:revive
```

Comment:
Why skip the check?

Reply:
`revive` check recommend us to fix structure name from `SocketContext` to `Context`
Because `SocketContext` contains package name `socket`, it's recommended to remove prefix `socket`

Reply:
Can we add a space like "// nolint:revive"? So keep same with other comments.

Reply:
fixed.